specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -9018,7 +9033,6 @@
       "properties": {
         "type": {
           "$ref": "#/definitions/OpenAuthenticationProviderType",
```

Comment:
Seems service backend would treat it as "AAD" by default. So we don't need to set it in request body, right? If yes, maybe this property could keep read-only. Does it make sense?